package.json

```diff
@@ -11,7 +11,7 @@
     "dashboarding"
   ],
   "private": true,
-  "version": "2.2.0",
+  "version": "2.2.1",
   "branch": "main",
```

Comment:
can you bump this to be `2.2`?

Reply:
Hey @kavilla you mean the `branch` to 2.2 as `"branch": "2.2",`?